WORKSPACE

```diff
@@ -135,12 +135,12 @@ python_register_toolchains(
 load("@rules_python//python:pip.bzl", "pip_parse")
 
 pip_parse(
-    name = "allwpilib_pip_deps",
+    name = "pip_deps",
```

Comment:
We don't name the `@maven` hub with a `_deps` suffix. We should be consistent here.

Also, would it be better to give this a tool-agnostic name, like `pypi`, in case either we or the ecosystem decide to stop using `pip`?

Reply:
All the other repos I've worked with end up calling this `pip_deps`.  https://rules-python.readthedocs.io/en/latest/pypi/download-workspace.html#download-workspace calls it `pip_deps`.  (AOS calls it `pip_deps` because that's what the docs do, and renaming that will have broader impact)

Reply:
Ah, I'm not consistent either.  The docs call it `my_deps`.